### PR TITLE
Implement refresh endpoint (as an admin endpoint)

### PR DESCRIPTION
**Pull request by skykanin**

### CODE REVIEW

**Comment by skykanin** [comment]
don't merge this yet, I still want to explore implementing unit tests for this

File: onyxia-api/src/main/resources/application.properties [hunk 1]
```diff
@@ -41,4 +41,6 @@ event.webhook.includes=
 # List of events types to ignore for the webhook. e.g service.uninstall,service.install
 event.webhook.excludes=
 # Response stream configuration
-spring.mvc.async.request-timeout=600000
\ No newline at end of file
+spring.mvc.async.request-timeout=600000
+# Enabled admin endpoints
+admin.enabled=true
```
Comment (olevitt) [inline onyxia-api/src/main/resources/application.properties:46]: We have to set this to `false` by default :)

**Review by olevitt — CHANGES_REQUESTED** [review]
Almost good to go, there is just an oversight in the default configuration

**Review by olevitt — APPROVED** [review]
LGTM, thank you ! :+1:

**Pull request merged**